### ISSUE TITLE
Preserving case in manual slugs?

### ISSUE THREAD

**Issue by funnelfiasco** [lektor/lektor#807]
(I'm opening a new issue so as to not distract from the main conversation in other issues)

Is it currently possible to preserve case in manual slugs (e.g. `_slug: AFDAFC-201208270030.html` as described in #795)? I see that slug case is preserved when not specifying a slug manually. Based on a [comment in 806](https://github.com/lektor/lektor/issues/806#issuecomment-678828370), I tried various contortions with and without the `lektor-slugify` plugin, but wasn't able to make it behave as desired. 

As far as I can tell, case preservation is not currently supported for manual slugs, but I wanted to ask directly because it's entirely possible that I've missed something that should be obvious.

**Comment by xlotlu** [comment]
> I see that slug case is preserved when not specifying a slug manually.

That is just ironic.

AFAIK there is no official stance on case preservation. What you described sounds like a bug on top of a mis-feature.

**Comment by funnelfiasco** [comment]
I took a pass at implementing the suggestion in #668 (a settable `slug_options` ...option), but my Python skills are not up to the task. For my own purposes, a patch that just adds `lowercase=False` to the `slugify()` call is sufficient (and within my abilities), so I'm unblocked for what I need.

I'll close this issue, since my question is answered, and let discussion of slug improvements continue in #668 and #806
